crates/command_palette/src/command_palette.rs

```diff
@@ -80,7 +80,7 @@ impl CommandPalette {
             previous_focus_handle,
         );
 
-        let picker = cx.new_view(|cx| Picker::new(delegate, cx));
+        let picker = cx.new_view(|cx| Picker::uniform_list(delegate, cx));
```

Comment:
Fixed this in the latest commit. The uniform list was not taking padding into account when scrolling to a specific item. 

I would advise against using `Picker::list` everywhere, as the uniform_list implementation is more performant. uniform_list can optimise the layout process (if the assumption that all items are the same height is true), by only measuring a single item and then multiplying the height by that. list can't do that, because items have different heights, therefore it needs to call measure on pretty much every item that is in view (bounds + overdraw). 
So I suggest keeping `Picker::uniform_list` everywhere except the recent projects picker.

TLDR: We should only be using `Picker::list` when there are items with different heights. I believe that recent projects is the only place were this is true right now.

Reply:
Looks like the `clippy` is unhappy now, so while we are at it, let's add a rustdoc with those considerations for both `Picker` constructor methods then?
It was not very obvious for me, so I assume more people will find the explanations useful.

Reply:
What is `clippy` complaining about? Im not able to see any logs in the linux CI. I added some documentation to `Picker::list` and `Picker::uniform_list`.

Reply:
Sorry, I did not look at the actual CI run, but seems that it failed due to out of disk space for the Linux task, so disregard that.

Thank you for the docs nonetheless.

Reply:
All good, looks like it is passing now.